kernel/signal.c

```diff
@@ -155,7 +155,7 @@ static long _default_signal_handler(unsigned signum)
             myst_tcall_wake(process_thread->event);
         }
     }
-    thread->exit_status = 0;
+    thread->exit_status = 128 + signum;
```

Comment:
What is the significance of 128?

Reply:
Can you use _NSIG here?

Reply:
The magic number is from here: https://www.gnu.org/software/bash/manual/html_node/Exit-Status.html. There seems to be no macro for it.